README.md

```diff
@@ -14,6 +14,43 @@ However, in cases where it is important to track documents on a milestone-by-mil
 
 Depending on the nature of the documents, publishers should consider duplicating information in the parent sections documents block also.
 
+## Worked Example
```

Comment:
Just use "Example" in extensions.

Reply:
Sorry, I followed the [lot extension](https://github.com/open-contracting-extensions/ocds_lots_extension) pattern